specification/_global/get/GetRequest.ts

```diff
@@ -163,8 +163,9 @@ export interface Request extends RequestBase {
      * A comma-separated list of stored fields to return as part of a hit.
      * If no fields are specified, no stored fields are included in the response.
      * If this field is specified, the `_source` parameter defaults to `false`.
-     * Only leaf fields can be retrieved with the `stored_field` option.
-     * Object fields can't be returned;​if specified, the request fails.
+     * Only leaf fields can be retrieved with the `stored_fields` option.
+     * Object fields can't be returned; if specified, the request fails.
+     * For an example, refer to [Retrieve stored fields](https://www.elastic.co/docs/reference/elasticsearch/rest-apis/retrieve-stored-fields).
```

Comment:
Sorry, I didn't mention it earlier, we can use `ext_doc_id` here. Please add the next line to the [table.csv](https://github.com/elastic/elasticsearch-specification/blob/main/specification/_doc_ids/table.csv) file as part of this PR, so we can keep track the links we are adding:
`retrieve-stored-fields,https://www.elastic.co/docs/reference/elasticsearch/rest-apis/retrieve-stored-fields`
and then refer to the ID as the suggestions shows below, please.
```suggestion
     * @ext_doc_id retrieve-stored-fields
```